easybuild/easyblocks/l/libxml2.py

```diff
@@ -95,9 +95,10 @@ def sanity_check_step(self):
         """Custom sanity check for libxml2"""
         shlib_ext = get_shared_lib_ext()
         pyfiles = ['libxml2mod.%s' % shlib_ext, 'libxml2.py', 'drv_libxml2.py']
+        #[os.path.join(self.pylibdir, x) for x in pyfiles],
         custom_paths = {
             'files': ["lib/libxml2.a", "lib/libxml2.%s" % shlib_ext] +
-                     [os.path.join(self.pylibdir, x) for x in pyfiles],
-            'dirs': ["bin", self.pylibdir, "include/libxml2/libxml"],
+                     [tuple([(os.path.join(y,x)) for y in self.all_pylibdirs]) for x in pyfiles],
```

Comment:
tiny style remark: space after `,` (in `(y,x)`)

and while you're at it, why not use more descriptive letter: `f` for file, `d` for directory; makes it easier to read imho

```python
[tuple([(os.path.join(d, f)) for d in self.all_pylibdirs]) for f in pyfiles],
```